tests/core/mgmstatemachinetest.cpp

```diff
@@ -14,12 +14,13 @@
 #include "../../src/core/funcbloc.h"
 #include "fbcontainermock.h"
 
+const SFBInterfaceSpec gcEmptyInterface = {};
```

Comment:
I would suggest making this a static constant in `CFunctionBlockMock` instead of a global variable.

Reply:
good point.